src/main.rs

```diff
@@ -74,7 +71,7 @@ fn main() -> Result<(), Box<dyn Error>> {
 
         if in_menu {
             // Input handlers for the menu
-            while event::poll(Duration::default())? {
+            while event::poll(Duration::from_millis(KEYBOARD_POLLING_TIMEOUT_MS))? {
```

Comment:
What is the rationale behind this change?

`Duration::default()` is zero seconds, which was chosen because I didn't want to introduce any delay reading the keyboard events. There is a separate `sleep` at the end of the loop aimed at reducing CPU usage.

A better improvement would be to change this to `Duration::ZERO` to be more explicit.

Reply:
Actually, I was also looking at a way to get a predictable frames-per-second rate. You are right that using Duration::ZERO would help that even more, than forcing some timeout value.

I guess I sill feel a little uncomfortable with the Defaults::default() concept. Will take some time for me to get used to  ;-). Still I tend to lean towards the Duration::ZERO way of naming this